doc/developers_guide.md

```diff
@@ -124,3 +124,36 @@ sonar-scanner -Dsonar.organization=exasol -Dsonar.host.url=https://sonarcloud.io
 To use a local, non-published version of the extension interface in integration tests, edit [pkg/integrationTesting/extensionForTesting/package.json](./../pkg/integrationTesting/extensionForTesting/package.json) and replace the version of `"@exasol/extension-manager-interface"` with the path to your local clone of [extension-manager-interface](https://github.com/exasol/extension-manager-interface).
 
 Make sure to not commit the modified `package.json`.
+
+## Extension Registry
+
+The extension registry is an HTTPS service that provides a JSON file containing links to all available extensions. It consists of an S3 Bucket and a CloudFront distribution deployed via CDK.
+
+### Initial Configuration
+
+1. Create file `registry/lib/config.ts` with the following content:
+    ```ts
+    export const CONFIG = {
+        owner: 'your.email@example.com'
+    }
+    ```
+2. Run `npm install`
+3. Configure AWS profile and region:
+    ```sh
+    export AWS_PROFILE=<profile>
+    export AWS_REGION=eu-central-1
+    ```
+
+### Deploy Changes
+
+Run `npm run cdk diff`. If the output looks good, run `npm run cdk deploy`.
+
+To get the output variables of the deployed stack (e.g. bucket name and CloudFront distribution host name), run the following command:
+
+```sh
+aws cloudformation describe-stacks --stack-name ExtensionManagerRegistry --query "Stacks[0].Outputs[].{key:ExportName,value:OutputValue}"
+```
+
+### Deploy Registry Content
+
+To deploy the content of the Extension Registry to `test` or `prod` stage, run script `AWS_PROFILE=<profile> upload-registry.sh test|prod`. This will upload one of the json files in the `content` folder to the S3 bucket and invalidate the CloudFront cache.
```

Comment:
```suggestion
To deploy the content of the Extension Registry to `test` or `prod` stage, run script `AWS_PROFILE=<profile> upload-registry.sh <prefix>` with `<prefix>` being either `test` or `prod`. This will upload the json file with the specified prefix into the `content` folder to the S3 bucket and invalidate the CloudFront cache.
```

Reply:
We could also call the variable `<stage>` and explain that this variable is also interpreted as a prefix of the respective file to upload.